wrappers/here.rb

```diff
@@ -326,8 +326,9 @@ def get(url_base, object, params = {})
                 return
               elsif additional_data.include?({ 'key' => 'error_code', 'value' => 'NGEO_ERROR_ROUTING_CANCELLED' })
                 return
-              elsif additional_data.include?({ 'key' => 'error_code', 'value' => 'NGEO_ERROR_ROUTE_NO_START_POINT' })
-                raise UnreachablePointError.new(error), "Here, UnreachablePoint: #{params.keys.grep(/waypoint/).map{|key| params[key]}}"
```

Comment:
If you are modifying this line, I think you should delete `test_router_no_route_point` because it cannot pass if there is no raise, it literally checks that.

Reply:
done.